index.html

```diff
@@ -68,6 +72,189 @@ <h1 class="primary-text">I'm Christian<br />Glad to see<br />you!</h1>
+              <li><span class="css">CSS</span></li>
+              <li><span class="js">JavaScript</span></li>
+            </ul>
+          </div>
+          <button>See project</button>
+        </article>
+        <article class="card-work">
+          <figure>
+            <img src="images/SnapshootPortfolio4.png" alt="work-img1" />
+            <figcaption><h2>Multi-post Stories</h2></figcaption>
+          </figure>
+          <div class="type">
+            <p class="title">CANOPY</p>
+            <p class="domain">Backend</p>
+            <p class="year">2015</p>
+          </div>
+          <p class="description">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <div class="languages">
+            <ul>
+              <li><span class="html">HTML</span></li>
+              <li><span class="css">CSS</span></li>
+              <li><span class="js">JavaScript</span></li>
+            </ul>
+          </div>
+          <button disabled>See project</button>
+        </article>
+      </section>
+      <section class="about">
+        <h1>About<br />Myself</h1>
+        <p class="content">
+          Hello I'm a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If you
+          like what you see and have a project you need coded, don't hestiate to
+          contact me.
+        </p>
+        <div class="social">
+          <p>let's connect</p>
+          <ul>
+            <li>
+              <a href="#"
+                ><img src="images/twitter).svg" alt="twitter-icon"
+              /></a>
+            </li>
+            <li>
+              <a href="#"
+                ><img src="images/Linkedinicon.svg" alt="linkdn-icon"
+              /></a>
+            </li>
+            <li>
+              <a href="#"
+                ><img src="images/monster.svg" alt="monster-icon"
+              /></a>
+            </li>
+            <li>
+              <a href="#"><img src="images/Vector.svg" alt="gitbubicon" /></a>
+            </li>
+            <li>
+              <a href="#"><img src="images/thumb.svg" alt="thumb" /></a>
+            </li>
+          </ul>
+        </div>
+        <button class="resume">Get my resume</button>
+        <div class="mastering">
+          <ul>
+            <li>
+              <p>languages <i class="fa-solid fa-v"></i></p>
+              <span class="icone"
+                ><img src="images/javascript.png" alt="js-icone" />
+                <span class="label">JavaScript</span>
+              </span>
+              <span class="icone"
+                ><img src="images/html.png" alt="html-icone" />
+                <span class="label">HTML</span></span
+              >
+              <span class="icone"
+                ><img src="images/css.png" alt="" />
+                <span class="label">CSS</span></span
+              >
```

Comment:
- [ ] As asked by the previous reviewer, You have not used the list tags in these. You have to use the `ul` tag for languages and the `li` tag for all the elements Javascript, HTML, CSS. Fix it.

Reply:
is that let's connect a title? my review for the first project was actually the one who told me to put that p right there